transports/sql/native-delayed-delivery.md

```diff
@@ -0,0 +1,49 @@
+---
+title: SQL Server Native Delayed Delivery
+summary: Describes the native delayed delivery implementation in the SQL Server transport
+reviewed: 2017-06-05
+component: SqlServer
+versions: '[3.1,)'
+---
+
+In Versions 3.1 and above, the SQL Server transport can take advantage of native [delayed delivery](/nservicebus/messaging/delayed-delivery.md) without then need to run the [timeout manager](/nservicebus/messaging/timeout-manager.md). Instead, the transport creates infrastructure which can delay messages using native SQL Server transport features. To enable the native delayed delivery use the following API:
+
+snippet: EnableNativeDelayedDelivery
+
+NOTE: In this mode the timeout manager will still be running in order to process all outstanding delayed messages. Refer to the [Disabling the timeout manager](/transports/sql/native-delayed-delivery.md#disabling-the-timeout-manager) section for details on how to disable the timeout manager entirely.
+
+## How it works
```

Comment:
Probably better as "configuration" header? I'd also link from here to the upgrade guide, but that can be done in the other PR (let's just make a note to not forget).